### PR TITLE
Update Generic Duration Picker (EXPOSUREAPP-5746)

### PR DESCRIPTION
**Pull request by jurajkusnier**
Update generic duration look and feel.
image

### CODE REVIEW

**Review by mtwalli — DISMISSED** [review]
lgtm

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/durationpicker/DurationExtension.kt [hunk 1]
```diff
@@ -3,18 +3,8 @@ package de.rki.coronawarnapp.ui.durationpicker
 import org.joda.time.Duration
 
 fun Duration.toContactDiaryFormat(): String {
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/durationpicker/DurationExtension.kt:5]: Maybe add tests to verify if this extension still works as expected?

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -1921,6 +1921,14 @@
     <string name="duration_dialog_ok_button">OK</string>
     <!-- NOTR -->
     <string name="duration_dialog_default_value">00:00</string>
+    <!-- XTXT: Duration hours text -->
+    <string name="duration_hours">"Stunden"</string>
+    <!-- XTXT: Duration minutes text -->
+    <string name="duration_minutes">"Minuten"</string>
+    <!-- XTXT: Duration Organize title -->
+    <string name="duration_typical_lenght_of_stay">"Typische Aufenthaltsdauer"</string>
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1929]: These last two Strings don't belong in this PR I guess.

**Review by LukasLechnerDev — DISMISSED** [review]
LGTM! 2 small remarks

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/durationpicker/DurationExtension.kt (hunk 1 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/durationpicker/DurationExtension.kt:5]: One more test is never a bad idea 😃

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/values-de/strings.xml]: yeah, they don't need to be here

**Review by mtwalli — APPROVED** [review]

**Review by LukasLechnerDev — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA reviewed
